src/App.js

```diff
@@ -1,19 +1,22 @@
 import "./App.css";
-import {Routes, Route} from 'react-router-dom'
-import { ErrorPage, Footer, Landing, NavBar, NotesPage } from "./imports/imports";
+import { Routes, Route } from "react-router-dom";
+import { ErrorPage, Footer, Landing, Login, NotesPage, Signup } from "./imports/imports";
```

Comment:
Instead of importing every component inside `imports.js` and then exporting,
You can just import every component from `components` folder by making an `index.js` file and exporting from there

Reply:
Okay will do that